ehrql/query_engines/in_memory_database.py

```diff
@@ -99,7 +99,7 @@ def __getitem__(self, name):
         return self.name_to_col[name]
 
     def to_records(self):
-        for p in self.patients():
+        for p in sorted(self["patient_id"].patients()):
```

Comment:
This can be `sorted(self.patients())`, I think - `self.patients()` just returns `self["patient_id"].patients()`